packages/dashboard/src/components/Wizard/Wizard.Base.tsx

```diff
@@ -22,20 +35,64 @@ export class WizardBase extends React.Component<IWizardProps, {}> {
     // if the step to render is already passed in, use that
     const wizardStepProps = this.props.stepToShow ? this.props.stepToShow : getStepToShow(this.props);
 
-    const classNames = getClassNames(this.props.styles!, {
+    const wizardStyleProps = {
       theme: this.props.theme!,
       isSubStep: wizardStepProps.isSubStep!,
-      isFirstSubStep: wizardStepProps.isFirstSubStep!
-    });
+      isFirstSubStep: wizardStepProps.isFirstSubStep!,
+      clickedForward: this.lastStepIndexShown <= wizardStepProps.index! ? true : false
+    };
+
+    // Update the step index showing
+    this.lastStepIndexShown = wizardStepProps.index!;
+
+    const classNames = getClassNames(this.props.styles!, wizardStyleProps);
+
+    const contentAnimKey = 'contentSectionAnim-' + wizardStepProps.id;
+    const contentSectionKey = 'contentSection-' + wizardStepProps.id;
+    const contentTitleKey = 'contentTitle-' + wizardStepProps.id;
+    const contentKey = 'content-' + wizardStepProps.id;
+
+    let mainStepTransitionClass = {
+      enter: classNames.stepSlideUpEnter,
+      enterActive: classNames.stepSlideUpEnterActive,
+      exit: classNames.stepSlideUpExit,
+      exitActive: classNames.stepSlideUpExitActive,
+      exitDone: classNames.stepSlideUpExitDone
+    };
+
+    if (!wizardStyleProps.clickedForward) {
```

Comment:
Let's write this as either a ternary or an if/else block since it's mutually exclusive.